ddp-workspace/projects/toolkit/src/lib/components/activity/workflow-start-activity.component.ts

```diff
@@ -2,105 +2,108 @@ import { ChangeDetectorRef, Component, Inject, OnDestroy, OnInit } from '@angula
+    WindowRef,
+    SessionStorageService
 } from 'ddp-sdk';
 import { filter, map, mergeMap, take } from 'rxjs/operators';
 
 @Component({
-  selector: 'toolkit-workflow-start-activity',
-  template: `
-      <toolkit-header [showButtons]="false"
-                      [stickySubtitle]="stickySubtitle">
-      </toolkit-header>
-      <ddp-activity *ngIf="show && instanceGuid"
-                    [studyGuid]="studyGuid"
-                    [activityGuid]="instanceGuid"
-                    (submit)="navigate($event)"
-                    (stickySubtitle)="showStickySubtitle($event)">
-      </ddp-activity>`
+    selector: 'toolkit-workflow-start-activity',
+    template: `
+        <toolkit-header [showButtons]="false"
+                        [stickySubtitle]="stickySubtitle">
+        </toolkit-header>
+        <ddp-activity *ngIf="show && instanceGuid"
+                      [studyGuid]="studyGuid"
+                      [activityGuid]="instanceGuid"
+                      (submit)="navigate($event)"
+                      (stickySubtitle)="showStickySubtitle($event)">
+        </ddp-activity>`
 })
 export class WorkflowStartActivityComponent implements OnInit, OnDestroy {
-  public studyGuid: string;
-  public instanceGuid: string;
-  public stickySubtitle: string;
-  public show = true;
-  public activityCode: string;
-  private anchor: CompositeDisposable = new CompositeDisposable();
+    public studyGuid: string;
+    public instanceGuid: string;
+    public stickySubtitle: string;
+    public show = true;
+    public activityCode: string;
+    private anchor: CompositeDisposable = new CompositeDisposable();
 
-  constructor(
-    private workflowBuilder: WorkflowBuilderService,
-    private temporaryUserService: TemporaryUserServiceAgent,
-    private session: SessionMementoService,
-    private workflow: WorkflowServiceAgent,
-    private windowRef: WindowRef,
-    private cdr: ChangeDetectorRef,
-    @Inject('ddp.config') private configuration: ConfigurationService,
-    @Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService) { }
+    constructor(
+        private workflowBuilder: WorkflowBuilderService,
+        private temporaryUserService: TemporaryUserServiceAgent,
+        private session: SessionMementoService,
+        private workflow: WorkflowServiceAgent,
+        private windowRef: WindowRef,
+        private cdr: ChangeDetectorRef,
+        private sessionStorageService: SessionStorageService,
+        @Inject('ddp.config') private configuration: ConfigurationService,
+        @Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService) { }
 
-  public ngOnInit(): void {
-    this.studyGuid = this.toolkitConfiguration.studyGuid;
-    this.fetchActivity();
-  }
+    public ngOnInit(): void {
+        this.studyGuid = this.toolkitConfiguration.studyGuid;
+        this.fetchActivity();
+        this.sessionStorageService.remove('nextUrl');
```

Comment:
the only thing which was changed in removing nextUrl value from session storage once user on prequalifier page, since there is no point to keep it if user creates a new account, but not login into existing one